app/javascript/mastodon/actions/notifications.js

```diff
@@ -21,6 +21,7 @@ export const NOTIFICATIONS_SET_BROWSER_PERMISSION = 'NOTIFICATIONS_SET_BROWSER_P
 
 defineMessages({
   mention: { id: 'notification.mention', defaultMessage: '{name} mentioned you' },
+  mentioned_you: { id: 'notification.mentioned_you', defaultMessage: '{name} mentioned you' },
```

Comment:
I dont think this solves the issue, it simply defines this key, but it is not used in the service worker

Reply:
See #34668